opm/input/eclipse/Schedule/UDQ/UDQContext.cpp

```diff
@@ -178,6 +178,23 @@ namespace Opm {
         };
     }
 
+    std::optional<double>
```

Comment:
this doesn't seem to be optional, I guess we are complying to some generic code somewhere?

Reply:
> this doesn't seem to be optional, I guess we are complying to some generic code somewhere?

Yeah, mostly.  In isolation, it doesn't make a lot of sense to return `optional<double>` here because region-level summary vectors are always available.  I chose the return type mostly for symmetry with the other [`get_*_variable()`](https://github.com/OPM/opm-common/blob/1617f713216af2f92880aacf818d35f6f72e7861/opm/input/eclipse/Schedule/UDQ/UDQContext.hpp#L66-L75) member functions at the field, group, and well levels.  In those cases, especially at the well level, it makes a little bit more sense to use `optional<double>` because the well might not have come online at the time of the query and `nullopt` would signify that fact.  I think I'd prefer to keep the `optional` here though, because it'd make **slightly** more sense when we get full support for region-level UDQs&ndash;i.e., summary vectors named `RU*` and we're going to convert to `optional<>` when calling [`UDQSet::assign()`](https://github.com/OPM/opm-common/blob/1617f713216af2f92880aacf818d35f6f72e7861/opm/input/eclipse/Schedule/UDQ/UDQSet.hpp#L354-L363) anyway.